src/components/Button/Button.stories.tsx

```diff
@@ -113,3 +115,20 @@ export const MultiLineText: Story = {
     </HStack>
   ),
 }
+
+export const OverrideStyles: Story = {
+  render: () => (
+    <>
+      <Text>
+        Show custom styling examples here for visual testing of overrides from
+        the theme config
+      </Text>
+      <VStack>
+        <IconButton aria-label="toggle" icon={<MdNightlight />} px="1.5" />
+        <ButtonLink to="#" borderRadius="full" px="0" py="0">
+          <Translation id="get-involved" />
+        </ButtonLink>
+      </VStack>
+    </>
+  ),
+}
```

Comment:
Use this story to compile any visual testing where styles are overridden on the component render. Can get wacky here with edge cases to ensure the overrides will work.